config/locales/client.en.yml

```diff
@@ -2792,59 +2797,59 @@ en:
       title: "Keyboard Shortcuts"
       jump_to:
         title: "Jump To"
-        home: "<b>g</b>, <b>h</b> Home"
-        latest: "<b>g</b>, <b>l</b> Latest"
-        new: "<b>g</b>, <b>n</b> New"
-        unread: "<b>g</b>, <b>u</b> Unread"
-        categories: "<b>g</b>, <b>c</b> Categories"
-        top: "<b>g</b>, <b>t</b> Top"
-        bookmarks: "<b>g</b>, <b>b</b> Bookmarks"
-        profile: "<b>g</b>, <b>p</b> Profile"
-        messages: "<b>g</b>, <b>m</b> Messages"
-        drafts: "<b>g</b>, <b>d</b> Drafts"
+        home: "%{shortcut} Home"
+        latest: "%{shortcut} Latest"
+        new: "%{shortcut} New"
+        unread: "%{shortcut} Unread"
+        categories: "%{shortcut} Categories"
+        top: "%{shortcut} Top"
+        bookmarks: "%{shortcut} Bookmarks"
+        profile: "%{shortcut} Profile"
+        messages: "%{shortcut} Messages"
+        drafts: "%{shortcut} Drafts"
       navigation:
         title: "Navigation"
-        jump: "<b>#</b> Go to post #"
-        back: "<b>u</b> Back"
-        up_down: "<b>k</b>/<b>j</b> Move selection &uarr; &darr;"
-        open: "<b>o</b> or <b>Enter</b> Open selected topic"
-        next_prev: "<b>shift</b>+<b>j</b>/<b>shift</b>+<b>k</b> Next/previous section"
+        jump: "%{shortcut} Go to post #"
+        back: "%{shortcut} Back"
+        up_down: "%{shortcut}/%{shortcut_2} Move selection &uarr; &darr;"
+        open: "%{shortcut} or %{shortcut_2} Open selected topic"
```

Comment:
This doesn't feel right to me. As a translator I could quite easily do a stupid thing and break the HTML.

`"%{shortcut_2} or %{shortcut} Open selected topic"` would result in `Enter</span> or <span dir="ltr">o Open selected topic`

Translations should only have one `%{shortcut}` interpolation key. That way translators can't break the HTML and we don't have to update translations when we decide to add or remove a shortcut.

Instead there should be a translation for shortcut alternatives: `shortcut_alternatives: %{shortcut1} or %{shortcut2}`
I wonder if we should split the next/previous and up/down shortcuts into two entries, otherwise we will need a translation for that as well if we want to stick with the "/" instead of "or".

Reply:
Wow, this is a very good point that I completely missed, thanks for raising it.

Just to make sure I understand correctly, the output of `I18n.t("shortcut_alternatives", ....)` should be wrapped in a `<span>` and then the whole thing will replace the `%{shortcut}` variable in `"%{shortcut} Open selected topic"`. Is this right?

Regarding splitting next/previous etc.: I think we should keep them the way they are right now; having an extra translation key for the `/` is not a huge problem, right?

Reply:
> the output of `I18n.t("shortcut_alternatives", ....)` should be wrapped in a `<span>` and then the whole thing will replace the `%{shortcut}` variable in `"%{shortcut} Open selected topic"`. Is this right?

Yes, exactly.

> having an extra translation key for the `/` is not a huge problem, right?

Correct. That's not a problem.

Reply:
Ok I've made those changes. Let me know if there is anything I missed. Thanks!